docs/cimpl6LanguageReference.md

```diff
@@ -78,70 +78,52 @@ Any periods in the _namespace_ should be replaced by dashes. For example, for a
+>**Note:** While it is best practice to use only lower case letters, the strict requirements only require the word to begin with a lowercase letter. The rest of the namespace allows for mixed casing and hyphens.
 
-CIMPL classes are conventionally defined in [PascalCase](http://wiki.c2.com/?PascalCase). For example:
+### Class Names
 
-```
-Dosage
-DetectionTime
-AdverseEvent
-Neutrophils100WBCNFrPtBldQnAutoCntLabObs
-```
+CIMPL classes are conventionally defined in [PascalCase](http://wiki.c2.com/?PascalCase), for example, `Dosage`, `DetectionTime`, and `Neutrophils100WBCNFrPtBldQnAutoCntLabObs`.
 
 >**Note:** Letters, numbers, and hyphens are allowed. However, class names must begin with an uppercase letter.
 
 ### Fully Qualified Names
 
-CIMPL does not support duplicate class names within a single namespace. However, the same name may occur in different namespaces. In the case of naming collisions due to an included namespace in [`Uses`](#Uses), you have to refer to the class by its fully qualified name (FQN). The FQN is a combination of the `Element` namespace and its declared name (through concatenation delimited by a period). For example, the `Element` _CancerCondition_ in the namespace _onco.core_ has the following FQN:
-
-```
-onco.core.CancerCondition
-```
+CIMPL does not support duplicate class names within a single namespace. However, the same name may occur in different namespaces. In the case of naming collisions due to an namespace included by a [`Uses`](#Uses) statement, you have to refer to the class by its fully qualified name (FQN). The FQN is a concatenation of the namespace name and the class name (delimited by a period). For example, the class `CancerCondition` in the namespace `onco.core` has the FQN `onco.core.CancerCondition`. 
 
 FQNs are only required in case of naming collisions between namespaces.
 
 ### Value Set Names
 
-Value Set names are by convention written in [PascalCase](http://wiki.c2.com/?PascalCase), with a suffix of "VS". For example:
+Value Set names are by convention written in [PascalCase](http://wiki.c2.com/?PascalCase), with a suffix of "VS", for example, `HomeEnvironmentRiskVS` and `LandmarkTypeVS`.
 
-```
-ValueSet: HomeEnvironmentRiskVS
-```
 >**Note:** Value Set names must begin with an uppercase letter.
 
 ### Local Code Names
 
-Within a locally defined value set, the individual codes (denoted by `#`) are by convention written in lowercase [snake_case](https://en.wikipedia.org/wiki/Snake_case) with an underscore between words. For example:
-```
-ValueSet:          HomeEnvironmentRiskVS
-#smoke_detectors   "No Smoke Detectors"
-#radiation         "Radon or Other Radiation Source"
-#swimming_pool     "Swimming Pool"
-```
+Within a locally defined value set, the individual codes (denoted by `#`) are by convention written in lowercase [snake_case](https://en.wikipedia.org/wiki/Snake_case) with an underscore between words, for example, `#no_smoke_detectors`, `#radon`, and `#swimming_pool`.
```

Comment:
Suggest changing `#radon` to `#radiation` in the text to match the example, or remove it as there are already two examples.

Reply:
ok

Reply:
I wanted to include that example because it defines its own local code system.